services/installer.sh

```
@@ -30,23 +30,46 @@ if ! getent passwd keylime >/dev/null; then
+fi
+
+mkdir -p /usr/share/keylime
+cp -a "$TPM_CERT_STORE_SRC" /usr/share/keylime/ || exit 1
 
-chown keylime:keylime -R /etc/keylime
-chown keylime:keylime -R /var/lib/keylime
-chown keylime:keylime -R /var/log/keylime
-chown keylime:keylime -R /var/run/keylime
+# install tmpfiles.d config for keylime directories
+mkdir -p /usr/lib/tmpfiles.d
+cp $BASEDIR/keylime-tmpfiles.conf /usr/lib/tmpfiles.d/keylime.conf
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)